src/libraries/Microsoft.Extensions.DependencyInjection/tests/Common/src/AssemblyTestLog.cs

```diff
@@ -1,306 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This and LoggedTestBase already deleted in @JunTaoLuo 's PR: https://github.com/dotnet/extensions/pull/3040.

I had moved Logging.Tests under Microsoft.Extensions.DependencyInjection/tests/Common/, they need to eventually move to Logging tests I think. that is why I am not including files under Common in the test csproj
I'll be manually applying similar changes and will resolve conflicts with his change when I'm bringing over new PRs later

Reply:
Resolved: Moved to Logging tests